powershell/autorest-configuration.md

```diff
@@ -12,13 +12,14 @@ modelerfour:
   additional-checks: false
   always-create-content-type-parameter: false
   always-seal-x-ms-enums: true
+  treat-type-object-as-anything: true
 ```
 
 > if the modeler is loaded already, use that one, otherwise grab it.
 
 ``` yaml !isLoaded('@autorest/modelerfour')
 use-extension:
-  "@autorest/modelerfour": "4.15.414"
+  "@autorest/modelerfour": "4.24.2"
```

Comment:
The latest version is 2.42.3. If there is no issue, please pin to the latest version.